### DISCUSSION TITLE
auto recording for outgoing connections

### DISCUSSION THREAD

**Discussion by itservicescscon** [rustdesk/rustdesk#9395] [General]
Hey Guys, we bought the Basic Pro Version mostly for the client configurator. However as the topic says we desperatly need a option for recording our outgoing RustDesk Support with our Clients. 
The option for the Clients to automatic record incoming connections is implemented but not on our side. 
We still need to press the record Button every time we would like to record!
 Can this one be implemented in the future or am i missing something and its already there?

appreciate the feedback
kind regards
cscon GmbH

**Comment by rustdesk** [comment]
https://github.com/rustdesk/rustdesk/discussions/5453
